grammars/javascript.cson

```diff
@@ -1096,7 +1080,11 @@
           '1':
             'name': 'meta.delimiter.method.period.js'
           '2':
-            'name': 'entity.name.function.js'
+            'patterns': [
```

Comment:
I think(?) that this can just be `'include': '#functions'`, without the extra `patterns` block.